package.json

```diff
@@ -1,46 +1,47 @@
 {
```

Comment:
Looks like indentation was changed on this file. It should be fixed for a better diff.